test/higgstautau_cfg_80X_Aug16.py

```diff
@@ -1359,10 +1359,10 @@
 
 #from ICAnalysis.MVAMETPairProducer.mvaPFMET_cff_leptons_74X import mvaMetPairs
 
-#if release in ['80XMINIAOD']:
-#  process.ak4PFJets.src = cms.InputTag("packedPFCandidates")
+if release in ['80XMINIAOD']:
+  process.ak4PFJets.src = cms.InputTag("packedPFCandidates")
```

Comment:
Can you revert this then? I'm still seeing the changes. Feel free to save your copy under another name if you need it